woocommerce/woocommerce-ios
# Orders tab: adapt to various screens sizes for iPad split view support in iOS 15

Closes: #6950 


### Description


While testing another PR using iPad split view, I noticed the orders tab isn't currently adjusting to different app widths when it's in split view with another app. This seems to be only an issue in iOS 15. After adding Auto Layout constraints to pin the child view controller in `OrdersSplitViewWrapperController`, the orders tab now adjusts its size to the parent/app's size. Maybe something with split view controller changed in iOS 15 🤷🏻‍♀️

### Testing instructions


- Launch the app in iPad on iOS 15
- Enable split view with another app like Safari
- In the app, go to the Orders tab --> the orders tab should adjust its width to the app width, you can check by scrolling up or tapping an order to view order details

- [x] @jaclync tested the steps in iOS 14.5 iPad simulator
- [x] @jaclync tested the steps in iOS 15.5 iPhone simulator

### Screenshots


before | after
-- | --
![Simulator Screen Shot - iPad Air (4th generation) - 2022-05-26 at 12 23 19](https://user-images.githubusercontent.com/1945542/170420223-cd5ad253-be6e-48b4-9c35-314bc2405a57.png) | ![Simulator Screen Shot - iPad Air (4th generation) - 2022-05-26 at 12 24 04](https://user-images.githubusercontent.com/1945542/170420235-99c0de1e-8c7f-48b3-b14c-be8acf9779bd.png)


---
